src/Mod/TechDraw/Gui/CommandAnnotate.cpp

```diff
@@ -97,8 +97,8 @@ CmdTechDrawLeaderLine::CmdTechDrawLeaderLine()
 {
     sAppModule      = "TechDraw";
     sGroup          = QT_TR_NOOP("TechDraw");
-    sMenuText       = QT_TR_NOOP("Add a line to a view");
-    sToolTipText    = QT_TR_NOOP("Add a line to a view");
+    sMenuText       = QT_TR_NOOP("Add Line to View");
+    sToolTipText    = sMenuText;
```

Comment:
I still suggest not making the tooltip the same as the menu text. Just leave it as it was originally. `QT_TR_NOOP(...)`

Reply:
vocx is right, tooltips and menu entries aren't the same thing, and in a perfect world the menu entry would be short and to the point and the tool tip would be more detailed.    

But I'm going to merge this anyway.